.circleci/config.yml

```diff
@@ -900,6 +900,10 @@ jobs:
           # some native-dependent tests fail because of the lack of native
           # headers on emsdk-bundled clang
           test_targets: "other skip:other.test_native_link_error_message"
+          # There are tests that using libraries
+          # that are not included by "./embuilder build ALL".  For example the
+          # different variant of libSDL2_mixer is used by test_sdl2_mixer_wav
+          frozen_cache: false
```

Comment:
Better not to do this.    Instead you can add the variant you are testing the variants list in sdl2_mixer.py.

Reply:
Thanks I removed this and used `ogg` format for a test case.